packages/design-system/src/components/MonthPicker/MonthPicker.tsx

```diff
@@ -99,7 +99,6 @@ export const MonthPicker = (props: MonthPickerProps) => {
     if (props.onChange) {
       props.onChange(event);
     }
-    console.log('isControlled ', isControlled);
```

Comment:
I was just [about to remove this](https://github.com/CMSgov/design-system/pull/2893)! Your version with a linter rule is better.